.travis.yml

```diff
@@ -1,219 +1,161 @@
 ---
-sudo: false
 dist: xenial
 language: ruby
 cache: bundler
 before_install:
-  - rm -f Gemfile.lock
+  - sh ./.travis/setup.sh
 script:
   - 'bundle exec rake $CHECK'
 matrix:
   fast_finish: true
   include:
-  - rvm: 2.1.9
-    bundler_args: --without system_tests development release
-    env: PUPPET_VERSION="~> 4.0" CHECK=test PARALLEL_TEST_PROCESSORS=12
   - rvm: 2.4.4
     bundler_args: --without system_tests development release
     env: PUPPET_VERSION="~> 5.0" CHECK=test
-  - rvm: 2.5.1
+  - rvm: 2.5.3
```

Comment:
seems the latest release reverted ruby to 2.5.1 according to https://puppet.com/docs/puppet/6.2/about_agent.html

Reply:
![panda_tableflip](https://user-images.githubusercontent.com/341866/52118144-d7e83980-2615-11e9-9d20-f368c5312f72.gif)

I will check why this got reverted...

Reply:
lulz, also realized this is a modulesync PR, so not the right place to address these changes

Reply:
na this is fine. Feedback is always welcome. I will merge this as is (if it turns green) and afterwards update modulesync_config repo.

Reply:
I'm not able to find a note about 2.5.1 in the changelog, nor a Jira issue. Also the latest rpm seems to include 2.5.3:

```
[root@4fb6828f1e15 /]# /opt/puppetlabs/puppet/bin/ruby -v
ruby 2.5.3p105 (2018-10-18 revision 65156) [x86_64-linux]
[root@4fb6828f1e15 /]# yum info puppet-agent
Loaded plugins: fastestmirror, ovl
Loading mirror speeds from cached hostfile
 * base: mirror.fra10.de.leaseweb.net
 * extras: mirror.ratiokontakt.de
 * updates: mirror.imt-systems.com
Installed Packages
Name        : puppet-agent
Arch        : x86_64
Version     : 6.2.0
Release     : 1.el7
Size        : 93 M
Repo        : installed
From repo   : puppet6
Summary     : The Puppet Agent package contains all of the elements needed to run puppet, including ruby, facter, and hiera.
URL         : https://www.puppetlabs.com
License     : See components
Description : The Puppet Agent package contains all of the elements needed to run puppet, including ruby, facter, and hiera.
            : 
            : Contains the following components:
            : cleanup
            : cpp-hocon 0.2.0
            : cpp-pcp-client 1.6.0
            : dmidecode 2.12
            : facter 3.12.3
            : hiera 3.5.0
            : leatherman 1.5.4
            : libwhereami 0.2.2
            : module-puppetlabs-augeas_core 1.0.1
            : module-puppetlabs-cron_core 1.0.0
            : module-puppetlabs-host_core 1.0.2
            : module-puppetlabs-mount_core 1.0.2
            : module-puppetlabs-scheduled_task 1.0.0
            : module-puppetlabs-selinux_core 1.0.1
            : module-puppetlabs-sshkeys_core 1.0.2
            : module-puppetlabs-yumrepo_core 1.0.3
            : module-puppetlabs-zfs_core 1.0.1
            : module-puppetlabs-zone_core 1.0.1
            : puppet 6.2.0
            : puppet-resource_api v1.6.3-7-g2419162
            : puppet-runtime 201901130
            : pxp-agent 1.10.4
            : runtime
            : shellpath 2015-09-18
            : virt-what 1.18
            : wrapper-script

[root@4fb6828f1e15 /]#
```

I assume the docs are simply wrong. I will create an issue for that.

Reply:
https://tickets.puppetlabs.com/browse/DOCUMENT-977